dotcom-rendering/src/web/components/FrontSection.stories.tsx

```diff
@@ -50,6 +50,9 @@ const Placeholder = ({
 	</div>
 );
 
+/** The stories below consistently have padContent set to false, even though the default value and the one which ends up rendering the component in the application is true.
+ * This is so the stories below can match up with the section stories which the FrontSection component is modeled after. */
```

Comment:
I fear this leads to having stories which look different to what is in production, which is of very little value.

Reply:
I agree; I wrote that reasoning at first that these stories wouldn't directly translate into prod, and the ones that do (e.g. the containers with cards in them) were working. But it's still not a satisfying solution, so been trying to find one!

Reply:
I would probably stick with what I did in the original PR and make the stories look as much like PROD as possible and remove `padContent` entirely.